spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -1,24 +1,42 @@
 == Jakarta Data Providers
 
-Jakarta Data providers implement repository interfaces, performing queries and other operations related to entities per the rules of the Jakarta Data specification. A Jakarta Data provider makes the repository implementation available to the application via dependency injection. The Jakarta Data specification defines the rules by which Jakarta Data providers must abide to ensure that multiple Jakarta Data providers are able to coexist in a system without interfering or overlapping on the same injection points.
+A Jakarta Data provider might come as an integrated component of a Jakarta EE container, or it might be a separate component that integrates with the Jakarta EE container via standard or proprietary SPIs.
+For example, a Jakarta Data provider might use a CDI portable extension to integrate with dependency injection.
```

Comment:
```suggestion
A Jakarta Data provider can use a CDI portable extension to integrate with dependency injection.
```

Reply:
Not sure about this change. I was trying to be illustrative here, not prescriptive.

Reply:
Sure, that's fine. I'm pretty sure we state this elsewhere, so we can leave this as you wrote it and it will be fine.  I'll mark this comment resovled.